sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 12.0.0-beta.2 (Unreleased)
+## 12.0.0-beta.3 (Unreleased):
+
+## 12.0.0-beta.2 (2020-10-06):
```

Comment:
Looking at azure-tables releases. All you have to do is update the date to reflect the date this package would be released.